src/NumericalAlgorithms/DiscontinuousGalerkin/Tags.hpp

```diff
@@ -19,6 +19,12 @@
 #include "NumericalAlgorithms/Spectral/Projection.hpp"
 #include "Options/Options.hpp"
 
+/// Functionality related to discontinuous Galerkin schemes
+namespace dg {
+// The classes below should be moved in the `dg` namespace. See issue:
+// https://github.com/sxs-collaboration/spectre/issues/2140
```

Comment:
@wthrowe requested this comment be removed in #1725, so please remove it here too

Reply:
In that PR the comment in question referred to the PR itself. Linking to issues is a practice we have been encouraging I think. Thoughts?

Reply:
Oh you referred to a comment I had not seen until now. I remove it then. Does that mean you think we shouldn’t link to issues from code @wthrowe @nilsdeppe @kidder? I see no harm in making the relevant issue easier to find from the code